lua/anywise_reg/handlers.lua

```diff
@@ -18,6 +18,17 @@ M.handle_action = function(prefix, operator, textobject)
     data.update_reg_data(reg_name, operator, textobject)
 end
 
+M.handle_paste_behind = function(prefix)
```

Comment:
I think it would be better to change `handle_paste` to take an `operator` argument (which will be `p` or `P`) and avoid some code duplication.